static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -709,7 +708,18 @@ function WidgetBuilder({
       if (!fieldStrings.length) {
         // The grouping was cleared, so clear the orderby
         newQuery.orderby = '';
+      } else if (widgetBuilderNewDesign && !newQuery.orderby) {
+        const isDescending = newQuery.orderby.startsWith('-');
```

Comment:
if `!newQuery.orderby` is true, doesn't `newQuery.orderby.startsWith('-');` always evaluate to false?

Reply:
Great point, I removed this and the next line and defaulted it to prefixing with `-`